atomic_reactor/plugins/post_tag_from_config.py

```diff
@@ -83,15 +137,9 @@ def get_component_name(self):
         return name
 
     def run(self):
-        tags = self.get_tags()
-
-        if tags:
-            name = self.get_component_name()
-            for i, tag_suffix in enumerate(tags):
-                tag = '{}:{}'.format(name, tag_suffix)
-                self.log.debug('Using additional tag: %s', tag)
-                self.workflow.tag_conf.add_primary_image(tag)
-                # Store modified name.
-                tags[i] = tag
+        if self.tags:
```

Comment:
Nit-pick, but I think this ought to be `if self.tags is not None`. If `tags=[]` is supplied, we should add no tags.